gsoc/gsoc-2023.md

```diff
@@ -0,0 +1,159 @@
+# Welcome to Google Summer of Code with JSON Schema!
+
+JSON Schema will be applying to 2023's Google Summer of Code Program as a mentoring Organizarion. In case of being accepted, we will receive applications within the timeframe allocated by GSoC. Please, visit the Program timeline [here](https://developers.google.com/open-source/gsoc/timeline) for more details.
```

Comment:
```suggestion
JSON Schema will be applying to 2023's Google Summer of Code Program as a mentoring organization. In case of being accepted, we will receive applications within the timeframe allocated by GSoC. Please, visit the Program timeline [here](https://developers.google.com/open-source/gsoc/timeline) for more details.
```